core/mondoo-azure-security.mql.yaml

```diff
@@ -1427,5 +1439,384 @@ queries:
         4. Select on `Email notifications` and ensure that the `Notify about alerts with the following severity (or higher)` setting is checked and set to `High`
     query: |
       azure.cloudDefender.securityContacts.all( alertNotifications["minimalSeverity"] == "High" && alertNotifications["state"] == "On" )
+  - uid: mondoo-azure-security-ensure-that-ssl-enabled-postgresql
+    title: Ensure SSL connection enabled for PostgreSQL Database Server
+    severity: 80
+    docs:
+      desc: |
+        All communications between the clients and the PostgreSQL server should be through SSL/TLS to add a layer of encryption to prevent any man in the middle attacks.
+      audit: |
+        __cnspec run__
+
+        To audit Microsoft Azure with `cnspec run`:
+      
+        it is possible also to use client-secret for app connection instead of the certificate. 
+
+        Run the following query:
+
+          ```bash
+          cnspec run azure -c "azure.postgresql.servers.all( properties["sslEnforcement"] == "Enabled" )" --certificate-path <*.pem> --tenant-id <tenant_id> --client-id <client_id>
+          ```
+
+        __cnspec shell__
+
+        To audit Microsoft Azure with `cnspec shell`:
+
+        1. Launch `cnspec shell`:
+
+          ```bash
+          cnspec shell azure --certificate-path <*.pem> --tenant-id <tenant_id> --client-id <client_id>
+          ```
+        
+        2. Run the following query:
+
+          ```mql
+          azure.postgresql.servers.all( properties["sslEnforcement"] == "Enabled" )
+          ```
+      remediation: |
+        ### Microsoft Azure Portal
+
+        To update using the Microsoft Azure portal:
+        1. Log in to the Microsoft Azure portal at https://portal.azure.com 
+        2. Go to Azure Database for `PostgreSQL server`
+        3. For each database, click on `Connection security`
+        4. In `SSL` settings, click on `ENABLED` to enforce SSL connections
+    query: |
+      azure.postgresql.servers.all( properties["sslEnforcement"] == "Enabled" )
+  - uid: mondoo-azure-security-ensure-that-ssl-enabled-latest-version-mariadb
+    title: Ensure SSL connection enabled for MariaDB Database Server with the latest version
+    severity: 80
+    docs:
+      desc: |
+        All communications between the clients and the mariadb server should be through SSL/TLS to add a layer of encryption to prevent any man in the middle attacks.
```

Comment:
```suggestion
        All communications between the clients and the MariaDB server should be through SSL/TLS to add a layer of encryption to prevent any man in the middle attacks.
```